tutorials/System.exit.md

```diff
@@ -0,0 +1,6 @@
+Exits the Runtime
+
+# Example
+```js
+fin.System.exit().then(() => console.log('Exiting.')).catch(err => console.log(err));
```

Comment:
• Maybe something like "Runtime exiting" or "Runtime exited" is clearer
• Maybe this: `.catch(console.error);`